src/tests/unit/components/question-editor/QuestionEditor.spec.jsx

```
@@ -0,0 +1,52 @@
+  onSave,
+  loading,
+  isQuizQuestion
+}
+const questionInputFieldMockValue = QuestionEditorProps.data.text
+const answerInputFieldMockValue = QuestionEditorProps.data.type
+describe('Testing slider with input ', async () => {
+  it('-> should renders question input field', () => {
+    const { getByDisplayValue } = render(
+      <QuestionEditor {...QuestionEditorProps} />
+    )
+    const questionInputField = getByDisplayValue(questionInputFieldMockValue)
```

ESLint

[error] 'questionInputField' is assigned a value but never used.

(no-unused-vars)